notesserver/highlight.py

```diff
@@ -39,13 +39,28 @@ def build_search_kwargs(self, *args, **kwargs):
             highlight_options = {
                 'fields': {
                     content_field: {'store': 'yes'},
-                }
+                    'tags': {'store': 'yes'},
+                },
             }
             if isinstance(highlight, dict):
                 highlight_options.update(highlight)
             res['highlight'] = highlight_options
         return res
 
+    def _process_results(
```

Comment:
@benmcmorran did you consider moving the tag highlighting closer to the client, rather than making it look like haystack/ES is doing it?  I've always considered the insertion of this kind of markup from the server a little ugly and anti-encapsulation, but in the case of highlighting match text within a larger body I can understand the value.  When you're rendering tags tho, isn't it just a simple 'if' statement to style them differently?